controllers/boilers.js

```diff
@@ -104,7 +104,7 @@ exports.findOneType = (req, res) => {
     .then((data) => {
       if (!data) {
         return res.status(404).send({
-          message: `Boiler with Type id ${req.params.type} does not exist`,
+          message: `Boiler with Type id ${req.params.type} was not exist`,
```

Comment:
cambiar por. --> message: `Boiler with Type id ${req.params.type} does not exist`,